deployment/docker-compose.yml

```diff
@@ -50,6 +50,7 @@ services:
             - ./logs/nginx:/logs
             # Allow nginx to look at the storage cache and to keep an s3 cache of its own
             - ./storage/cache:/caches/pkgserver
+            - ./storage/static:/caches/pkgserver/static
```

Comment:
This is fine, right? Or is it better to have
```yaml
            - ./storage/cache:/caches/pkgserver/cache
            - ./storage/static:/caches/pkgserver/static
```

Reply:
I think it's better to separate them.

Reply:
Ok, done.